src/components/InputBar/InputBar.jsx

```diff
@@ -23,9 +20,7 @@ const mapDispatchToProps = dispatch => ({
 
 });
 
-const InputBar = connect(
+export default connect(
```

Comment:
nil--an opinionated change - I just felt like it would reduce the complexity of writing a component + have one less variable to toss around